src/tls13.c

```diff
@@ -1078,19 +1169,20 @@ int DeriveResumptionPSK(WOLFSSL* ssl, byte* nonce, byte nonceLen, byte* secret)
             return BAD_FUNC_ARG;
     }
 
-    PRIVATE_KEY_UNLOCK();
 #if defined(WOLFSSL_TICKET_NONCE_MALLOC) &&                                    \
     (!defined(HAVE_FIPS) || (defined(FIPS_VERSION_GE) && FIPS_VERSION_GE(5,3)))
+    PRIVATE_KEY_UNLOCK();
```

Comment:
Will the MAXQ work with TLS v1.3 when `WOLFSSL_TICKET_NONCE_MALLOC` is set? Consider documenting this.